### PR TITLE
Assignment2 @schuywalker

### PR DESCRIPTION
**Pull request by tremorris1999**
See [DEPLOYMENT.md](https://github.com/tremorris1999/EWU-CSCD379-2022-Spring/blob/assignment2/DEPLOYMENT.md) for deployment info.

### CODE REVIEW

File: wordle-web/components/game-board.vue [hunk 1]
```diff
@@ -1,13 +1,28 @@
 <template>
-  <v-card width="450">
+  <v-card class="ma-5 py-1" max-width="500">
```
Comment (DakotaCondos) [inline wordle-web/components/game-board.vue:2]: There is supposed to be a gradient background here for the game board. try using `style="gradient code here"`

File: wordle-web/components/keyboard.vue [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <template>
-  <v-card class="my-5 pa-5">
-    <v-row v-for="(charRow, i) in chars" :key="i" no-gutters justify="center">
-      <v-col v-for="char in charRow" :key="char" cols="1">
-        <v-container class="text-center">
+  <v-card class="ma-0 pa-0" color="transparent" flat>
```
Comment (DakotaCondos) [inline wordle-web/components/keyboard.vue:2]: There is supposed to be a gradient background here for the game board. try using `style="gradient code here"` as part of the v-card

File: wordle-web/components/settings-dialog.vue [hunk 3]
```diff
@@ -66,7 +66,7 @@ export default class SettingsDialog extends Vue {
   }
 
   turnOffTheLights() {
-    // Implement Me
+    this.$vuetify.theme.dark = true
   }
```
Comment (DakotaCondos) [inline wordle-web/components/settings-dialog.vue:71]: 2 more color themes need to be added here

File: wordle-web/layouts/default.vue [hunk 4]
```diff
@@ -1,33 +1,50 @@
...
     </v-main>
-    <v-footer app>
+    <v-navigation-drawer v-model="rightDrawer" :right="right" temporary fixed>
+      <v-list>
+        <v-list-item>
+          <v-btn text block>
+            <SettingsDialog />
+          </v-btn>
+        </v-list-item>
+
```
Comment (DakotaCondos) [inline wordle-web/layouts/default.vue:25]: need to add links to home and game page here

**Review by DakotaCondos — COMMENTED** [review]
review left with in line comments. a few items are missing but overall looks good

File: wordle-web/components/PrerollAd.vue [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  <v-card width="800" height="800">
+    <v-container class="overflow-hidden" fluid>
+      <v-row justify="center">
+        <v-col>
+          <!-- <Adsense
+            data-ad-client="ca-pub-1431122584842002"
+            data-ad-slot="9479195735"
+          ></Adsense> -->
+          <!-- That doesn't work without a top-level domain, anyway. Here's a fake ad instead -->
+          <a href="https://youtu.be/dQw4w9WgXcQ" target="_blank">
```
Comment (mmwoodfo) [inline wordle-web/components/PrerollAd.vue:11]: So I sent this link to Grant, and he immediately calls me dying of laughter. This was very, very funny. Awesome job guys!

**Review by mmwoodfo — COMMENTED** [review]
Honestly, this is really great. Good job guys!

**Pull request closed without merging**